pkg/chains/signing.go

```diff
@@ -189,6 +196,9 @@ func (o *ObjectSigner) Sign(ctx context.Context, tektonObj objects.TektonObject)
 				if err := b.StorePayload(ctx, tektonObj, rawPayload, string(signature), storageOpts); err != nil {
 					logger.Error(err)
 					merr = multierror.Append(merr, err)
+				} else {
+					logger.Debugf("Recording count of stored payload for %v", obj)
```

Comment:
Trace level?

Reply:
Removed the loggers as trace level  is not available.